package.json

```
@@ -23,5 +23,18 @@
     "eslint-plugin-react-refresh": "^0.4.19",
     "globals": "^15.15.0",
     "vite": "^6.2.0"
-  }
+  },
+  "description": "",
```

```suggestion
}
[
  {
      "_id": "68185f3ba2f0cd0010b20b02",
      "message": "Sunshine!",
      "hearts": 3,
      "createdAt": "2025-05-05T06:48:27.239Z",
      "__v": 0
  },
  {
      "_id": "681640df990c8100108a69c9",
      "message": "test <3",
      "hearts": 0,
      "createdAt": "2025-05-03T16:14:23.104Z",
      "__v": 0
  },
  {
      "_id": "68127ded8e19b90010a991f1",
      "message": "fgfhgghg",
      "hearts": 5,
      "createdAt": "2025-04-30T19:45:49.282Z",
      "__v": 0
  },
  {
      "_id": "6810ae1ee69088001032752f",
      "message": "Hello from the other side\n",
      "hearts": 0,
      "createdAt": "2025-04-29T10:46:54.003Z",
      "__v": 0
  },
  {
      "_id": "680be69a86814000101f64d0",
      "message": "helloooo\n",
      "hearts": 0,
      "createdAt": "2025-04-25T19:46:34.994Z",
      "__v": 0
  },
  {
      "_id": "680795b68ba03300101ca995",
      "message": "what's up",
      "hearts": 9,
      "createdAt": "2025-04-22T13:12:22.208Z",
      "__v": 0
  },
  {
      "_id": "6803f286c02ed10010d6af3d",
      "message": "hello",
      "hearts": 2,
      "createdAt": "2025-04-19T18:59:18.065Z",
      "__v": 0
  },
  {
      "_id": "67f6cfbda40e6f001088a6c4",
      "message": "Just booked my summer vacation 🌻",
      "hearts": 0,
      "createdAt": "2025-04-09T19:51:25.413Z",
      "__v": 0
  },
  {
      "_id": "67f6cf7fa40e6f001088a6c3",
      "message": "Spring is finally here!",
      "hearts": 5,
      "createdAt": "2025-04-09T19:50:23.385Z",
      "__v": 0
  },
  {
      "_id": "67cf0c5b169c370010fdc9fe",
      "message": "the weather is so nice\n",
      "hearts": 2,
      "createdAt": "2025-03-10T15:59:23.856Z",
      "__v": 0
  },
  {
      "_id": "67cf0934169c370010fdc9fd",
      "message": "sunshine\n",
      "hearts": 0,
      "createdAt": "2025-03-10T15:45:56.038Z",
      "__v": 0
  },
  {
      "_id": "67cee9976bfd180010c3a19f",
      "message": "the sun is shining\n",
      "hearts": 1,
      "createdAt": "2025-03-10T13:31:03.960Z",
      "__v": 0
  },
  {
      "_id": "67c0606e7ded3b0010ea324a",
      "message": "learning new things\n",
      "hearts": 0,
      "createdAt": "2025-02-27T12:54:06.950Z",
      "__v": 0
  },
  {
      "_id": "67c0606e7ded3b0010ea3249",
      "message": "learning new things\n",
      "hearts": 4,
      "createdAt": "2025-02-27T12:54:06.097Z",
      "__v": 0
  },
  {
      "_id": "67bcae996f4b600010016369",
      "message": "Waffles",
      "hearts": 0,
      "createdAt": "2025-02-24T17:38:33.169Z",
      "__v": 0
  },
  {
      "_id": "67bc4da05b0ec2001087ed53",
      "message": "Silly insta reels",
      "hearts": 0,
      "createdAt": "2025-02-24T10:44:48.130Z",
      "__v": 0
  },
  {
      "_id": "67bba4ca33b4d50010b6b564",
      "message": "Hej hej från aspudden ",
      "hearts": 1,
      "createdAt": "2025-02-23T22:44:26.016Z",
      "__v": 0
  },
  {
      "_id": "67bba17f33b4d50010b6b563",
      "message": "My daughter",
      "hearts": 2,
      "createdAt": "2025-02-23T22:30:23.899Z",
      "__v": 0
  },
  {
      "_id": "67bb5b994a29c00010acc556",
      "message": "uuuu there is a limit nice",
      "hearts": 1,
      "createdAt": "2025-02-23T17:32:09.221Z",
      "__v": 0
  },
  {
      "_id": "67bb5a654a29c00010acc555",
      "message": "Is there a character limit here I wonder?\n",
      "hearts": 2,
      "createdAt": "2025-02-23T17:27:01.215Z",
      "__v": 0
  }
]
{
  "message": "My happy thought"
}
```

---

Yes the character limit 140 characters.
```suggestion
  "description": "",
```